### ISSUE TITLE
Text Alignment is not persistent, always defaulted to right justified

### ISSUE THREAD

**Issue by Trikenstein** [flameshot-org/flameshot#3160]
### Flameshot Version

Flameshot v11.0.0 (11.0.0-2 Debian)
Compiled with Qt 5.15.2
linux: 5.15.0-69-generic
ubuntu: 22.04.2 (Gnome 42.5)

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

ubuntu: 22.04.2 (Gnome 42.5)

### Description

The text tool is always right-justified. Clicking on the "Tool Settings" at the left edge of the screen allows to set the text alignment to left justified. But when Flameshot is re-opened (ie. clicking on the Flameshot icon on the indicator icon bar (top right of the screen in Gnome). The text alignement is defaulted back to right-justified.

Flameshot settings screen has no option to set the default text-justification.

### Steps to reproduce

. Click on the flameshot icon
. Select an area
. Click on T icon
. Type a few line of text
. Click on the T icon or another tool icon
. The N lines of text are transformed from left justified to right justified

### Screenshots or screen recordings

![Flameshot_text_right-justified](https://user-images.githubusercontent.com/72353094/230830170-44008f65-3e8d-4ebc-8f7e-c7373ad1c8ad.png)

### System Information

- ubuntu: 22.04.2 (Gnome 42.5, X11)
- dual monitors, 2560x1440

```shell
xrandr --listactivemonitors
Monitors: 2
 0: +*DisplayPort-0 2560/597x1440/336+0+0  DisplayPort-0
 1: +DisplayPort-1 2560/597x1440/336+2560+0  DisplayPort-1
```
```shell
lshw -class display
WARNING: you should run this program as super-user.
  *-display                 
       description: VGA compatible controller
       product: Oland GL [FirePro W2100]
       vendor: Advanced Micro Devices, Inc. [AMD/ATI]
       physical id: 0
       bus info: pci@0000:04:00.0
       logical name: /dev/fb0
       version: 00
       width: 64 bits
       clock: 33MHz
       capabilities: vga_controller bus_master cap_list rom fb
       configuration: depth=32 driver=radeon latency=0 resolution=2560,1440
       resources: irq:115 memory:e0000000-efffffff memory:f0100000-f013ffff ioport:3000(size=256) memory:f0160000-f017ffff
```

**Comment by Trikenstein** [comment]
Not sure why the there is an indication saying that I have labelled "Unconfirmed Bug". This is REAL BUG. Very easy to reproduce. Just use the text tool type a few lines and the bug will show up. As shown in the screenshot.

**Comment by mmahmoudian** [comment]
> Not sure why the there is an indication saying that I have labelled "Unconfirmed Bug".

Because it is what we have define to assign to newly created issues by default. It will change when and if we manage to reproduce it

> This is REAL BUG. Very easy to reproduce. Just use the text tool type a few lines and the bug will show up. As shown in the screenshot.

This *WAS* a real bug. It was fixed in #2448 . You have installed a version that is already 3 versions old!
